Chart.yaml

```diff
@@ -30,7 +30,5 @@ maintainers:
     email: reinier@skoef.nl
   - name: nielshojen
     email: niels@hojen.net
-  - name: Pupperware Team
```

Comment:
Was that intentional?

Reply:
Yes, since the team does not seem to exist in the repo, linting the chart errors out on it. So choices are to either create it within the org/repo or remove it.